js/src/common/classes/GAClient.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Internal dependencies
+ */
+import { debounce } from '../helpers';
+
+/**
+ * Genesis Analytics Client
+ *
+ * Forked from BMO's work in Genesis Blocks.
+ *
+ * Follows the singleton pattern to prevent multiple instances of the GA Client from being used.
+ * https://developers.google.com/analytics/devguides/collection/gtagjs
+ */
+export default class GAClient {
+	/**
+	 * Is Google Analytics enabled.
+	 *
+	 * @type {boolean}
+	 */
+	enabled = false;
+
+	/**
+	 * Google Analytics Client
+	 *
+	 * @type {Object}
+	 */
+	client;
+
+	/**
+	 * Google Analytics Measurment ID.
+	 *
+	 * Todo: update this for GCB.
+	 *
+	 * @type {string}
+	 */
+	GA_ID = 'UA-12345';
+
+	/**
+	 * Class constructor.
+	 */
+	constructor() {
+		this.client = function() {
+			// @ts-ignore
+			window.dataLayer.push( arguments );
+		};
+
+		// @ts-ignore
+		this.config = window.genesisAnalyticsConfig || {};
```

Comment:
Does this value need to be `window.gcbAnalyticsConfig` ?

Reply:
Uf, you're right.

Reply:
Fixed in https://github.com/studiopress/genesis-custom-blocks/pull/101/commits/87d1f456f42c97069065698e45448101c43215b7